install_pim/installation_docker.rst

```diff
@@ -0,0 +1,214 @@
+
+
+Using the Docker images
+-----------------------
+
+Every flavor (dev or standard, community or enterprise) come with a Docker Compose file template, ready to be used. All you need is to copy it as ``docker-compose.yml``.
+
+If you intend to run behat tests, create on your host a folder ``/tmp/behat/screenshots`` (or anywhere else according to your compose file) with full read/write access to your user.
+Otherwise ``docker-compose`` will create it, but only with root accesses. Then failing behats will be unable to create reports and screenshots.
+
+
+Run and stop the containers
+---------------------------
+
+**All "docker-compose" commands are to be ran from the folder containing the compose file.**
+
+To start your containers, just run:
+
+.. code-block:: bash
+
+   $ docker-compose up -d
+
+To stop the containers, run:
+
+.. code-block:: bash
+
+   $ docker-compose stop
+
+but if you want to completely remove everything (containers, networks and volumes), then run:
+
+.. code-block:: bash
+
+   $ docker-compose down -v
+
+This, of course, will not delete the Akeneo application you cloned on your machine, only the Docker containers.
+
+
+Install and run Akeneo
+----------------------
+
+Configure Akeneo
+****************
+
+First, make sure that Akeneo database settings are as the containers expect.
+As you can see below, the ``database_host`` is the name of your MySQL service in the compose file.
+For Elasticsearch, ``index_hosts`` is the association of the login and password (``elastic`` and ``changeme``, respectively) of the container,
+the service name in the compose file (``elasticsearch``) and the output port of Elasticsearch (``9200``).
+
+.. code-block:: yaml
+
+   # /host/path/to/you/pim/app/config/parameters.yml
+   parameters:
+       database_driver: pdo_mysql
+       database_host: mysql
+       database_port: null
+       database_name: akeneo_pim
+       database_user: akeneo_pim
+       database_password: akeneo_pim
+       locale: en
+       secret: ThisTokenIsNotSoSecretChangeIt
+       index_name: akeneo_pim
+       index_hosts: 'elastic:changeme@elasticsearch:9200'
+
+.. code-block:: yaml
+
+   # /host/path/to/you/pim/app/config/parameters_test.yml
+   parameters:
+       database_driver: pdo_mysql
+       database_host: mysql-behat
+       database_port: null
+       database_name: akeneo_pim
+       database_user: akeneo_pim
+       database_password: akeneo_pim
+       locale: en
+       secret: ThisTokenIsNotSoSecretChangeIt
+       installer_data: PimInstallerBundle:minimal
+       index_name: akeneo_pim_behat
+       index_hosts: 'elastic:changeme@elasticsearch:9200'
+
+For more details on Akeneo configuration, don't hesitate to look at [Akeneo official documentation on the subject](https://docs.akeneo.com/latest/developer_guide/installation/index.html).
+
+
+Install Akeneo
+**************
+
+Now, you can initialize Akeneo, by simply running:
```

Comment:
please avoid the "just", "simply" etc, even if it's obvious when you masterize it, for newbie, especially when you discover something, it's always hard (even if straightforward and well documented :wink: )